src/components/AdminSettings.vue

```diff
@@ -161,6 +161,144 @@
 				</div>
 			</div>
 		</div>
+		<div class="managecd-projectfolders-and-password">
+			<FormHeading index="4"
+				:is-managed-project-heading="true"
+				:title="t('integration_openproject', 'Managed project folders (recommended)')"
+				:is-managed-folder-in-active="isManagedFolderInActive"
+				:is-group-folder-setup-correct="isGroupFolderStatusCorrect"
+				:is-complete="isManagedProjectFolderCompleted"
+				:is-disabled="isManagedGroupFolderSetUpInDisableMode"
+				:show-managed-folder-main-error="state.app_password_set" />
+			<div v-if="showDefaultManagedFolders">
+				<div v-if="isManagedGroupFolderSetUpFormInEdit">
+					<NcCheckboxRadioSwitch type="switch" :checked.sync="isGroupfolderSetupAutomaticallyReady" @update:checked="changeGroupFolderSetUpState">
+						<b>Automatically managed folders</b>
+					</NcCheckboxRadioSwitch>
+					<div v-if="isGroupfolderSetupAutomaticallyReady === false" class="complete-without-groupfolders">
+						<p class="managed-folder-description">
+							{{
+								t('integration_openproject', 'We recommend using this functionality but is not mandatory. Please activate in case you want to use the automatic creation and management of project folders.')
+							}}
+						</p>
+						<div class="form-actions">
+							<NcButton type="primary"
+								data-test-id="complete-without-projectfolders-form-btn"
+								@click="completeIntegrationWithoutGroupFolderSetUp">
+								<template #icon>
+									<CheckBoldIcon :size="20" />
+								</template>
+								{{
+									t('integration_openproject', iskeepCurrentCompleteWithoutIntegration)
+								}}
+							</NcButton>
+						</div>
+					</div>
+					<div v-else>
+						<p class="managed-folder-description">
+							{{
+								t('integration_openproject', 'Let OpenProject create folders per project automatically. It will ensure that every team member has always the correct access permission.')
```

Comment:
This should be permissions according to figma
```suggestion
								t('integration_openproject', 'Let OpenProject create folders per project automatically. It will ensure that every team member has always the correct access permissions.')
```